api/internal/core/entity/entity.go

```diff
@@ -304,3 +304,12 @@ type StreamRoute struct {
 	UpstreamID interface{}            `json:"upstream_id,omitempty"`
 	Plugins    map[string]interface{} `json:"plugins,omitempty"`
 }
+
+// swagger:model SystemConfig
+type SystemConfig struct {
+	ConfigName string                 `json:"config_name"`
+	Desc       string                 `json:"desc,omitempty"`
+	Payload    map[string]interface{} `json:"payload,omitempty"`
```

Comment:
does `Payload` need to be a map? maybe some configurations are string, float, or something?

Reply:
map is more appropriate for the configuration, and it is easier to add more items later on